GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java

```diff
@@ -264,9 +264,10 @@ public int compare(String o1, String o2) {
         }
     }
 
-    private Workbook createWorkbookAndFormats(){
+    private Workbook createWorkbookAndFormats() {
+        // This window may be too small for some OPTION questions
+        // on a comprehensive stats sheet
         Workbook wb = new SXSSFWorkbook(100);
```

Comment:
Maybe extracting 100 to a constant with a meaningful name would help understanding what it stands for